chain/chain/src/test_utils.rs

```diff
@@ -1417,7 +1417,7 @@ impl ChainGenesis {
         ChainGenesis {
             time: Clock::utc(),
             height: 0,
-            gas_limit: 1_000_000,
+            gas_limit: 10u64.pow(12),
```

Comment:
Did you mean to set this to 1Tgas? Or did you mean `10u64.pow(15)` for 1000 Tgas?

Reply:
Need to think about this! With 10^6, this breaks a whole bunch of tests (understandbly -- it's impossible to send a tx). So I've set it to *something*. 1Pgas is probably better choice, will think about this if this is the direction we'll go in.